test/test.setup.js

```diff
@@ -2,4 +2,4 @@
 
 process.env.NODE_ENV = 'test';
 
-jest.setTimeout(15000);
+jest.setTimeout(30000);
```

Comment:
Is extending the timeout strictly necessary?  Or is this perhaps left over from debugging issues when getting tests to work?

Reply:
its more of a CI issue, i can take a lot of resources to spawn things up and to run, just giving it enough time. 30s is often used everywhere so I am bringing it in line